Wave1_data/3.cellprofiling/scripts/run_cellprofiler_analysis.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform segmentation and feature extraction for each plate using CellProfiler Parallel
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import sys
+import time
+
+sys.path.append("../../../utils/")
+import cp_parallel
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# set the run type for the parallelization
+run_name = "analysis"
+
+# set main output dir for all plates
+output_dir = pathlib.Path("../analysis_output")
+output_dir.mkdir(exist_ok=True, parents=True)
+
+# directory where images are located within folders
+images_dir = pathlib.Path("../../2.illumination_correction/illum_directory/")
+
+# path to plugins directory as one of the pipelines uses the RunCellpose plugin
+plugins_dir = pathlib.Path(
+    "/home/lippincm/Documents/CellProfiler-plugins/active_plugins"
+)
+path_to_pipeline = pathlib.Path("../pipelines/analysis_5ch.cppipe").resolve(strict=True)
+
+
+# ## Create dictionary with all info for each well
+
+# In[3]:
+
+
+# get all directories with raw images
+dict_of_runs = {}
+raw_directories = list(images_dir.rglob("*"))
+raw_directories = [x for x in raw_directories if x.is_dir()]
+# filter for directories with images
+raw_directories = [x for x in raw_directories if len(list(x.glob("*.tiff"))) > 0]
+# #####################################
+# # for testing purposes
+# raw_directories = raw_directories[:2]
+# #####################################
```

Comment:
Same comment as previous PR for if you need to keep this part for now or remove in another PR later.

Reply:
removed